### DISCUSSION TITLE
Is 96kHz or higher sampling frequency tested with stm32l4 series?

### DISCUSSION THREAD

**Discussion by umichxuwentao** [hathach/tinyusb#1510] [General]
Hi Dear USB developers!

I would like to ask whether Full Speed 96kHz or higher sampling frequency has been tested for the current dwc2 driver with stm32l4 series or not, thanks a lot for answering my question!

**Comment by umichxuwentao** [comment]
96kHz and 192kHz are possible to implement.

**Comment by pigrew** [comment]
ISO endpoints are not implemented for STM32L4x2 and STM32L4x3.

It should work on STM32L4x5 and STM32L4x6, but I'm not sure if it's been tested or not.

With full-speed USB, you will need to be careful about bandwidth at higher sampling rates.
